geps/gep-1709/index.md

```diff
@@ -555,7 +508,81 @@ Creating a pull request to add the `ConformanceReport` will start the
 
 [crd]:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
 
-#### Certification Process
+### Reports submission
+
+Implementers can create as many reports as they want, with different API versions,
+API channels, modes, and implementation versions. The reports can be submitted upstream
+by creating a pull request to the Gateway API repository. The reports folder tree
+is the following:
+
+```console
+|-- conformance/reports
+|   |-- v1.0
+|   |   |-- acme-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v2.13-plain-report.yaml
+|   |-- v1.1
+|   |   |-- acme-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v2.14-plain-report.yaml
+|   |   |   |-- standard-v2.14-with-the-lot-report.yaml
+|   |   |   |-- extended-v2.14-with-the-lot-report.yaml
+|   |   |-- umbrella-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v1.8-plain-report.yaml
+```
+
+The main folder `conformance/reports` contains a set of sub-folders, each for a
+Gateway API version. Implementors will create their project folder in all the Gateway
+API version folders for which they have a supported implementation. The implementors
+folder contains the following set of files:
+
+* `README.md`: contains general information about the implementation and a table
+  with some information, as shown below, to give users a quick overview of
+  the implementation details.
+* `<API-channel>-<implementation-version>-<mode-name>-report.yaml`: the report
+  generated by the conformance test suite for the specific mode.
+
+#### README.md structure
+
+The README.md files SHOULD be structured in the following way (enrichment from
+implementors is allowed, even though the minimum content is described below):
+
+```md
+# Acme
+
+General information about the Acme/operator project
+
+## Support Matrix
+
+| API channel | Implementation version | Mode | Profiles | Extended Features | Report |
+|-------------|------------------------|------|----------|-------------------|--------|
+|             |                        |      |          |                   |        |
+|             |                        |      |          |                   |        |
+|             |                        |      |          |                   |        |
+
+```
```

Comment:
I'm not sure that a table will scale well here, especially for implementations that have a very long list of extended features. It also feels like it's really just duplicating information from conformance reports which could be difficult to keep in sync.

I think I'd prefer modeling this file more closely off of what upstream, like https://github.com/cncf/k8s-conformance/tree/master/v1.29/k3s for example - a brief description of the implementation + instructions for how to recreate.

Reply:
> I'm not sure that a table will scale well here, especially for implementations that have a very long list of extended features. It also feels like it's really just duplicating information from conformance reports which could be difficult to keep in sync.

yes, It makes sense, we'll end up having duplicate information from reports. Nevertheless, I think this kind of table would be pretty useful as a table of contents. What do you think about getting rid of the `profiles` and `extended features` columns? this way we have a single line for each report, with a quick link; something like this:

| API channel | Implementation version | Mode | Report |
|-------------|------------------------|-------------------|--------|
|             |                        |      |          |
|             |                        |      |          |

> I think I'd prefer modeling this file more closely off of what upstream, like https://github.com/cncf/k8s-conformance/tree/master/v1.29/k3s for example - a brief description of the implementation + instructions for how to recreate.

👍  I've added a "To reproduce" section in the README.md

Reply:
I think that's great, thanks!